tests/iso/smoke.sh

```
@@ -0,0 +1,247 @@
+[[ "$timeout_seconds" =~ ^[1-9][0-9]*$ ]] || fail "QEMU_TIMEOUT_SECONDS must be a positive integer"
+command -v "$qemu_binary" >/dev/null || fail "qemu-system-x86_64 is not installed"
+command -v xorriso >/dev/null || fail "xorriso is not installed"
+
+if ! xorriso -indev "$iso_path" -report_el_torito plain 2>&1 | grep -q "El Torito"; then
+    fail "ISO has no readable El Torito boot catalog"
+fi
+
+mkdir -p "$runtime_directory"
+"$qemu_binary" \
+    -name iso-smoke-test \
+    -machine ${qemu_machine},accel=${qemu_accel} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)